renku/core/models/datasets.py

```diff
@@ -985,6 +985,7 @@ class ImageObjectRequestJson(marshmallow.Schema):
     file_id = marshmallow.fields.String()
     content_url = marshmallow.fields.String()
     position = marshmallow.fields.Integer()
+    download = marshmallow.fields.Bool(default=False)
```

Comment:
Should we hint the type through better naming here? Maybe `is_downloaded` is better suited?

Reply:
`is_downloaded` would be a bit misleading, as it's more a `should_be_downloaded`, i.e. no download has happened at the time the request is sent. Maybe `do_download`, `force_download` or `should_be_downloaded` ?

Reply:
I named it `mirror_locally`. Couldn't really think of a name that's not overly verbose, like `should_be_mirrored_to_repository` and I think this should be at least a bit more clear than just `download`.